migrations/scripts/upgrade-to-lts.sh

```diff
@@ -482,7 +407,7 @@ function main {
   echo "Done! Please contact Astronomer support if any issues are detected."
   echo
   echo "Please install the new CLI:"
-  echo "curl -sSL https://install.astronomer.io | sudo bash -s -- v0.16.1"
+  echo "curl -sSL https://install.astronomer.io | sudo bash -s -- v0.23.4"
   echo
   echo "You may choose to upgrade Airflow versions by changing your Dockerfile, for example:"
   echo "FROM quay.io/astronomer/ap-airflow:1.10.10-alpine3.10-onbuild"
```

Comment:
you may want to change it to the most recent 2.0.x build of airflow and point to debian image instead of alpine as we are no longer building alpine images

Reply:
Done.